documentation/LLC-quickstart.md

```diff
@@ -0,0 +1,361 @@
+Getting Started - Generate the RLC rest-level client libraries.
+================================================================
+# Prerequisites
+You may refer to this [link](https://github.com/Azure/azure-sdk-for-js/blob/main/CONTRIBUTING.md#prerequisites) for the environment set up prerequisites in azure-sdk-for-js repository. 
+
+# How to generate RLC
+We are working on to automatically generate everything right now, but currently we still need some manual work to get a releasable package. Here're the steps of how to get the package.
+1. **Create a swagger/README.md file.**  
+    We are using autorest to generate the code, but there's a lot of command options and in order to make the regenerate process easier in the cases of refresh the rest api input or change the code generator version, you need to document the generate command parameters.  
+    Here's an example of the swagger/README.md
+    ~~~
+    
+    # Azure Purview Catalog TypeScript Protocol Layer
+    
+    > see https://aka.ms/autorest
+    ## Configuration
+    
+    ```yaml
+    package-name: "@azure-rest/purview-account"
+    title: PurviewAccount
+    description: Purview Account Client
+    generate-metadata: false
+    license-header: MICROSOFT_MIT_NO_VERSION
+    output-folder: ../
+    source-code-folder-path: ./src
+    input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/purview/data-plane/Azure.Analytics.Purview.Account/preview/2019-11-01-preview/account.json
+    package-version: 1.0.0-beta.1
+    rest-level-client: true
+    add-credentials: true
+    credential-scopes: "https://purview.azure.net/.default"
+    use-extension:
+      "@autorest/typescript": "6.0.0-beta.14"
+    ```
+    ~~~ 
+    Here, we need to replace the value in `package-name`, `title`, `description`, `input-file`, `package-version`, `credential-scopes` into **your own service's** `package-name`, `title`, `description` etc.   
+    
+    ---
+    **NOTE**
+    
+    It's always recommended to replace the version of code generator @autorest/typescript with the latest version you can find in [npmjs.com](https://www.npmjs.com/package/@autorest/typescript) in latest tag.  
+    
+    **For the first time, you will need to change the generate-metadata value as true, so that it can generate the basic package.json, tsconfig.json file for you. After the first generation, you can set it back as false as we have some manual changes in this file and don't want them get overwrite by generated ones.**   
+    
+    ---  
+  
+1. **edit rush.json**  
+    As the libraries in this azure-sdk-for-js repository are managed by rush, you need to add an entry in rush.json under projects section to make sure it works. For example:
+    ```
```

Comment:
When the service has already been onboarded to js sdk previously, I don't  think they need to add the configuration to rush.json